### ISSUE TITLE
Forced Revert opening Win11 System>Display menu

### ISSUE THREAD

**Issue by Abominables** [dylanraga/win11hdr-srgb-to-gamma2.2-icm#50]
Win11 23H2
Problem: conversion reverts
When: Opening System > Display Menu

**Comment by yunuset** [comment]
I don't think this can be fixed, as Windows defaults to the original values when the display menu is opened. I would like to hear too if anyone finds a permanent solution.
